File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    # Sahil Gupta
    def twoSum(self, nums, target):
        num_map = {}   # dictionary to store number and its index
        
        for i in range(len(nums)):
            complement = target - nums[i]
            
            if complement in num_map:
                return [num_map[complement], i]
            
            num_map[nums[i]] = i

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> myarr;
        
        for(int i=0; i<nums.size(); i++){
            for(int j=1; j<nums.size(); j++){
                if(i!=j){
                    if((nums[i]+ nums[j]) == target ){
                        myarr.push_back(i);
                        myarr.push_back(j);
                        break;
                    }
                }
            }
            
        }
        myarr.resize(2);
        return myarr ;
    }
};


    // for(int j=1; j<nums.size(); j++){
                
    //             if((nums[i]+ nums[j]) == target){
    //                 myarr.push_back(i);
    //                 myarr.push_back(j);
    //             }
                
    //         }